tests/rptest/services/redpanda_cloud.py

```diff
@@ -82,6 +87,15 @@ class CloudTierName(Enum):
     GCP_5_P5 = 'tco-p5-tier-5-gcp'
     GCP_6_P5 = 'tco-p5-tier-6-gcp'
     GCP_7_P5 = 'tco-p5-tier-7-gcp'
+    AZURE_1 = 'tier-1-azure-v1-x86'
```

Comment:
Shouldn't these be "v2"?

https://github.com/redpanda-data/cloudv2/blob/296f2e807d61b61f09b8add258fa377f2b9036f7/install-pack/24.1.yml#L4002

Reply:
Yes, that was my original commit before going to vacation. We now have new Tier names (v2) and Chris is working on another topic branch with more improvements. So this is a bit outdated now. Might close it and then he will merge everything together.